src/data/icons/svg/SOHM.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 25.4.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 1000 1000" style="enable-background:new 0 0 1000 1000;" xml:space="preserve">
+<style type="text/css">
```

Comment:
hello thanks for the contribution.
We don't accepts any <style>  tag or style attribute within a svg icons.
These svg files are parsed in order for them to be consumed by our live app on both mobile and desktop and have to follow a strict format to do so.
try limiting the icon to a single path and a single color aswell to make it flat. :smile:

Reply:
@LFBarreto we've updated the tokens & added a new token, WSOHM.

Reply:
I also added WSOHM token address via the form here: https://developers.ledger.com/docs/token/erc20-bep20/#2-how-to-get-my-token-listed